modules/rostests/apitests/advapi32/RtlEncryptMemory.c

```diff
@@ -9,26 +9,117 @@
 
 #include <ntsecapi.h>
 
-START_TEST(RtlEncryptMemory)
+static const CHAR TestData[32] = "This is some test Message!!!";
+
+void TestEncrypt(ULONG OptionFlags)
 {
-    static const CHAR TestData[32] = "This is some test Message!!!";
-    CHAR Buffer[32];
+    CHAR DECLSPEC_ALIGN(8) Buffer[32];
     NTSTATUS Status;
 
     /* Size must be aligned to 8 bytes */
-    Status = RtlEncryptMemory(Buffer, 7, RTL_ENCRYPT_OPTION_SAME_PROCESS);
+    RtlCopyMemory(Buffer, TestData, sizeof(Buffer));
+    Status = RtlEncryptMemory(Buffer, 7, OptionFlags);
```

Comment:
```suggestion
    Status = RtlEncryptMemory(Buffer, RTL_ENCRYPT_MEMORY_SIZE + 1, OptionFlags);
```

Reply:
Come on @SergeGautherie .... he wants to test **7 bytes**, but now you are changing the size to something else....

Reply:
Changed, though `7` or `9` should make no difference.